views/bootswatch.pug

```diff
@@ -13,15 +13,15 @@ block content
                         - if (i === helpers.theme.selected(config, theme))
                             button.btn.btn-primary(type="button", disabled) Active!
                         - else
-                            button.btn.btn-primary(type="button", onclick="tryIt(" + i + ")") Try it!
+                            button.btn.btn-primary(type="button", data-theme=i) Try it!
```

Comment:
My thought in doing it though client js was supporting static site generation down the road. However, at this point, I think I would prefer to put the entire site behind a CDN, which would work nicely with the method you're suggesting.

I'll leave it up to you, if you want to play with it on the short term, otherwise, I might be able to look at tweaking it in a few weeks.